src/placeholder/empty.tsx

```diff
@@ -0,0 +1,187 @@
+
+import type * as React from "react";
+
+import type { RecipeVariantProps } from "../../styled-system/css";
+import type { IconProps } from "../common/Icon";
+
+/**
+ * Defines the styles for the Empty component using the sva (styled variant API) pattern.
+ * This creates a multi-slot component with different visual states and variants.
+ *
+ * Slots:
+ * - root: Main container with centered content
+ * - iconContainer: Wrapper for the icon and its decorative rings
+ * - ringContainer: Individual decorative rings around the icon
+ * - iconWrapper: Container for the icon with background
+ * - icon: The actual icon with color manipulation
+ * - content: Container for text content
+ * - title: Heading text
+ * - description: Supporting text
+ *
+ * Variants:
+ * - normal: Default state with brand colors
+ * - danger: Error state with danger colors
+ */
+const emptyStyles = sva({
+  slots: ["root", "iconContainer", "ringContainer", "iconWrapper", "icon", "content", "title", "description"],
+  base: {
+    root: {
+      position: "relative",
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      justifyContent: "center",
+      minHeight: "80",
+      width: "full",
+      height: "full",
+      textAlign: "center",
+      overflow: "hidden",
+    },
+    iconContainer: {
+      position: "relative",
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+      mb: "4",
+    },
+    ringContainer: {
+      position: "absolute",
+      borderRadius: "full",
+      borderWidth: "1",
+      borderColor: "border.primary",
+      pointerEvents: "none",
+      zIndex: "0",
+    },
+    iconWrapper: {
+      padding: "3",
+      borderRadius: "full",
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+    },
+    icon: {
+      zIndex: "1",
+      boxShadow: "1",
+      objectFit: "contain",
+      position: "relative",
+      flexShrink: "0",
+      display: "block",
+    },
+    content: {
+      transform: "translateY(-150px)",
+      paddingX: "6",
+    },
+    title: {},
+    description: {
+      mt: "1",
+      maxWidth: "80",
+      textWrap: "balance",
+      mx: "auto",
+    },
+  },
+  variants: {
+    variant: {
+      normal: {
+        iconWrapper: {
+          bg: "bg.secondary._alt",
+        },
+        icon: {
+          filter: "[brightness(0) saturate(100%) drop-shadow(0px 1000px 0 var(--colors-bg-brand-primary))]",
+          transform: "translateY(-1000px)",
+        },
+      },
+      danger: {
+        iconWrapper: {
+          borderColor: "fg.danger.primary",
+        },
+        icon: {
+          filter: "[brightness(0) saturate(100%) drop-shadow(0px 1000px 0 var(--colors-fg-danger-primary))]",
+          transform: "translateY(-1000px)",
+        },
+      },
+    },
+  },
+  defaultVariants: {
+    variant: "normal",
+  },
+});
+
+type EmptyStyleProps = RecipeVariantProps<typeof emptyStyles>;
```

Comment:
Why isn't the recipe added to our preset?

Reply:
I thought we would have this for only ppi but we've discussed to reuse it in every project.

In short, I"m gonna move to preset